<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Валюты</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QPushButton" name="val_1">
          <property name="text">
           <string>Доллар</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="val_2">
          <property name="text">
           <string>Евро</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="val_3">
          <property name="text">
           <string>Фунт стерлингов</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="val_4">
          <property name="text">
           <string>Швейцарский франк</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
